src/main/kotlin/org/openrewrite/RecipeMarkdownGenerator.kt

```diff
@@ -959,7 +959,7 @@ class RecipeMarkdownGenerator : Runnable {
                         {% tab title="Maven Command Line" %}
                         {% code title="shell" %}
                         ```shell
-                        mvn -U org.openrewrite.maven:rewrite-maven-plugin:run \
+                        ./mvnw -U org.openrewrite.maven:rewrite-maven-plugin:run \
                           -Drewrite.recipeArtifactCoordinates=${origin.groupId}:${origin.artifactId}:LATEST \
                           -DactiveRecipes=${recipeDescriptor.name}
```

Comment:
We're a bit inconsistent with the use of `-Drewrite.` versus leaving out that `rewrite.` prefix, and I only have myself to blame. Shall we standardize on with or without `rewrite.` prefix if you're here anyway?

Reply:
I would say standardize on having the prefix. Less likely to collide with some other plugin that way

Reply:
Thanks for the clarification. Is this correct then:

```shell
mvn -U org.openrewrite.maven:rewrite-maven-plugin:run \
  -Drewrite.recipeArtifactCoordinates=${origin.groupId}:${origin.artifactId}:LATEST \
  -Drewrite.activeRecipes=${recipeDescriptor.name}
```

Reply:
That seems to work here indeed; thanks for making that change as well

Reply:
Awesome - appreciate the feedback. These changes should be reflected the next time we do an OpenRewrite release.